### PR TITLE
dont pass our logger in to tapestry

### PR DESCRIPTION
**Pull request by kevinhodges**
#### What does this PR do? (please provide any background)
Stops passing The Works logger into Tapestry, we don't use the logs so i'm removing stuff we push to Loggly.

#### What tests does this PR have?
No additional tests.

#### How can this be tested?

Fire up https://github.com/holidayextras/the-works/pull/1153 locally and run something like http://localhost:3000/hotelEventProducts/THTOLY_CBRBIE

#### Any tech debt?
No

#### What gif best describes how you feel about this work?

- I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

By approving a review you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

---

### CODE REVIEW

File: lib/pluginTapestry.js [hunk 1]
```diff
@@ -186,8 +186,6 @@ const register = function (server, pluginOptions) {
   // where to log messages
   var config = server.methods.getService('tapestry')
   config.cache = server.methods.getConfig().cache
-  // We've seen requests logging ~500Kb to loggly with the Tapestry logging turned on, this when multiplied by the 22k packageRates requests, is filling up loggly and providing no value. Please carefully consider before enabling the logger.
-  config.logger = server.methods.getDataLoggingWrapper('tapestry')
```
Comment (JordanHood) [inline lib/pluginTapestry.js]: didn't @jask84 add this in for his remove HXCM for tapestry move work?

Comment (kevinhodges) [reply lib/pluginTapestry.js]: Looking at the comments we did turn it on for that project, it's not currently providing value though.

Comment (JordanHood) [reply lib/pluginTapestry.js]: ok 👍

Comment (jask84) [reply lib/pluginTapestry.js]: spoke with Kev about this, we send more useful logs from the-works now anyway - https://github.com/holidayextras/the-works/blob/staging/flows/tapestryService.js#L77

**Pull request merged**